other-docs/guides/launching-a-site-on-altis.md

```diff
@@ -0,0 +1,126 @@
+# Launching a Site on Altis
+
+The process for launching a site on Altis varies depending on the type of project. Nonetheless, there are common steps that need to be done no matter the context and this guide will walk you through what needs to be done in preparation for your site launch.
+
+## Planning
+
+Create a launch issue in the project repository. This issue should include:
+
+* Client name
+* Client business region
+* Client contact
+* Launch date
+* Feature freeze date
+* Code freeze date
+* Content freeze date (if migrating an existing site)
+* Team members needed for the deploy
+* Domains hosted on Altis
+* Domain(s) used for email
+* FROM address for each site in the network
+* GitHub repository
```

Comment:
If the issue is going on the project's own repo this should be implicit too.

Reply:
That might not be obvious if we're telling them to make the GitHub issue to gather information and then submit a Zendesk ticket from the Altis Dashboard. If we're not seeing this issue in GitHub, do you think we should still pull out the repo link?